tests/socket-sentto-recvfrom-ipv6-udp.phpt

```diff
@@ -0,0 +1,64 @@
+--TEST--
+Test if socket_recvfrom() receives data sent by socket_sendto() via IPv6 UDP
+--CREDITS--
+Copied from php/php-src and adjusted, originally created by 
+Falko Menge <mail at falko-menge dot de>
+PHP Testfest Berlin 2009-05-09
+--SKIPIF--
+<?php
+if (substr(PHP_OS, 0, 3) == 'WIN') {
+	die('skip Not valid for Windows');
```

Comment:
This is valid if compiled with `--enable-ipv6`. The AppVeyor configure options might need some adjustments.

Reply:
Can you adjust AppVeyor to support IPV6?

Reply:
Sure, I'll create a PR. Also, some of these tests aren't working correctly (see https://travis-ci.org/krakjoe/pthreads/jobs/334677874#L800)

Reply:
Looks like a travis issue https://github.com/travis-ci/travis-ci/issues/8711

Reply:
For the time being fixed with eaa0a589f10138fae2cd47c4dd227f30b655c095